docs/README.md

```diff
@@ -1,154 +1,257 @@
 # Docker Documentation
 
-The source for Docker documentation is here under `sources/` and uses extended
-Markdown, as implemented by [MkDocs](http://mkdocs.org).
+The source for Docker documentation is in this directory under `sources/`. Our
+documentation uses extended Markdown, as implemented by
+[MkDocs](http://mkdocs.org).
 
-The HTML files are built and hosted on
-[http://docs.docker.com](http://docs.docker.com), and update automatically
-after each change to the `docs` branch of [Docker on
-GitHub](https://github.com/docker/docker) thanks to post-commit hooks.
+Through the use of post-commit hooks, Docker builds and updates our static HTML
+documentation automatically after each change to the `docs` branch of [this
+repository](https://github.com/docker/docker). You can find the resulting
```

Comment:
parse error on the first sentence. I'm guessing you want a comma between `and updates` and `our static HTML`, and I think there's a word missing in-between `documentation automatically` - perhaps `regenerates`?

I assume that (today) this is an aspirational paragraph?